gci/views.py

```diff
@@ -28,4 +30,26 @@ def index(request):
                  '<a href="https://github.com/{username}">{username}</a>'
                  .format(student_url=student_url, student_id=student_id,
                          username=username))
+    timestamp = datetime.now().strftime('%Y-%m-%d %H:%M:%S')
+    s.append('</ul><i id="time" data-time="{unix}">'
```

Comment:
sprinkle with CSS classes so that other ppl can make it prettier without fiddling here.